.travis.yml

```diff
@@ -24,8 +25,8 @@ before_install:
 
 before_script:
 
-    - "sh -e /etc/init.d/xvfb start"
-    - "export DISPLAY=:99.0"
+     #- "sh -e /etc/init.d/xvfb start"
+     #- "export DISPLAY=:99.0"
     - "wget http://selenium-release.storage.googleapis.com/2.43/selenium-server-standalone-2.43.1.jar"
```

Comment:
Old version of selenium, newest one is 3.141.59 :)
https://www.seleniumhq.org/download/